src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -518,7 +518,11 @@ private function createEncoder($config, ContainerBuilder $container)
         // Argon2i encoder
         if ('argon2i' === $config['algorithm']) {
             if (!Argon2iPasswordEncoder::isSupported()) {
-                throw new InvalidConfigurationException('Argon2i algorithm is not supported. Please install the libsodium extension or upgrade to PHP 7.2+.');
+                if (\defined(PHP_VERSION_ID) && \PHP_VERSION_ID >= 70200) {
```

Comment:
`\defined(PHP_VERSION_ID)` is broken. Passing an integer to `defined` does not make sense.

Working code would be `\defined('PHP_VERSION_ID')`, but this is useless, as it is *always* defined in PHP 5.2+ (and we don't care about 5.1)